packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -19,6 +29,10 @@ export class ResourceConstants {
         AuthCognitoUserPoolLogicalID: 'AuthCognitoUserPool',
         AuthCognitoUserPoolNativeClientLogicalID: 'AuthCognitoUserPoolNativeClient',
         AuthCognitoUserPoolJSClientLogicalID: 'AuthCognitoUserPoolJSClient',
+
+        // Relational Database
+        RelationalDatabaseDataSource: 'RelationalDatabaseDataSource',
```

Comment:
Can we move these constants into the new package?

Reply:
done